docs/components/multiselect.md

```diff
@@ -836,6 +799,7 @@ const allItems = new Array(10).fill().map((_, i) => ({
 }));
 
 export default defineComponent({
+  components: { KongIcon },
```

Comment:
question: should this section be script setup?

Reply:
I wanted to convert it, but it's rather large and after 20 painful minutes of trying to do it painlessly I decided to leave it for later. All the examples in this doc are using script setup though